components/raftstore/src/store/compaction_guard.rs

```diff
@@ -86,7 +102,12 @@ pub struct CompactionGuardGenerator<P: RegionInfoProvider> {
     use_guard: bool,
     // The boundary keys are exclusive.
     boundaries: Vec<Vec<u8>>,
+    next_level_boundaries: Vec<Vec<u8>>,
```

Comment:
Add a comment to descritbe it's L+2 level(input L, output L+1). And what's that if L=5